### PR TITLE
fix/updated-debug-support-message

### PR DESCRIPTION
**Pull request by kyegupov**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

The previous error message was not very helpful and the suggested command is actually problematic to run. Let's simplify it.

#### Screenshot

image

### CODE REVIEW

**Comment by adrukh** [comment]
I'll chime in as well. The reason for this extremely verbose debug message is in order to help our users help themselves. We implicitly rely on the actual dev tool being available and being able to build the project we are testing. Our users don't always know that. We used to receive more than a few support tickets on CLI failures rooted in this assumption not being communicated properly. We found this debug message to be very relevant in both communicating our dependency on a real dev env for the project, as well as a very good guardrail to lead our users to successfully running our CLI on the project.

While I agree the message becomes more convoluted with the temp file being part of it, the reason behind it holds and needs to be considered. Let's not https://en.wikipedia.org/wiki/Don%27t_throw_the_baby_out_with_the_bathwater please.

**Comment by kyegupov** [comment]
@adrukh the thing is, the relevance of the error message was lost when we switched from parsing the dependency tree to using our own injected script. Now, the error handler actually displays accumulated stdout/stderr texts, so we have no need to run the script again. A new kind of error message is needed and it has arrived....

@lili2311 today, only today, for the price of one PR you are getting:
[x] debug output including the actual command ran, error code, stdout and stderr
[x] output of `gradle -v`
[x] clear instructions for the end users of what to send us
[x] vivid ANSI coloring of the output
[x] `failure-states` tests actually running again!

**Comment by adrukh** [comment]
You have a point indeed. I will point out that even though we could do much with the actual output, I believe the user's benefit was in seeing the exact gradle error message and fixing it themselves so that gradle runs properly.

I have no objection to this change, just stressing out the usability factor behind it. We are all on the receiving end of the support cases that may or may not follow, and we all have our finger on the release button if we want to improve this further. :)

**Comment by adrukh** [comment]
All this lovely ansi escape codes - how will they behave in limited CI envs? :)

**Comment by kyegupov** [comment]
@adrukh we were always displaying the Gradle error message and we are still doing it :)

The ANSI colorings in CIs are likely to be stripped, but I can add a check for the terminal type just in case...

File: lib/index.ts [hunk 1]
```diff
@@ -144,10 +144,9 @@ async function getAllDeps(root, targetFile, options, subProject): Promise<DepTre
     };
   } catch (error) {
     error.message = error.message + '\n\n' +
-      'Please make sure that `' + command + ' ' + args.join(' ') +
-      '` executes successfully on this project.\n\n' +
-      'If the problem persists, collect the output of `' +
-      command + ' ' + args.join(' ') + '` and contact support@snyk.io\n';
+      'Please ensure you have calling the snyk command with correct arguments.\n' +
```
[the remark this answers is not in the document]
Comment (kyegupov) [reply lib/index.ts]: Fixed.

**Comment by lili2311** [comment]
A screenshot of the changes would be nice please :) How does the CLI look like using this?

**Comment by kyegupov** [comment]
image

**Pull request merged**